Libraries/PeriphDrivers/Include/MAX32662/adc.h

```diff
@@ -202,6 +202,7 @@ typedef struct {
     mxc_adc_avg_t avg_number; ///< no of samples to average
     mxc_adc_div_lpmode_t lpmode_divder; ///< Divide by 2 control in lpmode
     uint8_t num_slots; ///< num of slots in the sequence
+    int8_t dma_channel; ///< User should call MXC_DMA_AcquireChannel() to assign DMA Channel number
```

Comment:
I would suggest changing this comment to indicate that the field is ignored for non-DMA transactions.
```C
///<The channel to use for DMA-enabled transactions.  Users should call \ref MXC_DMA_AcquireChannel to assign the DMA channel number.  For non-DMA transactions, this field is ignored.
```

It looks like we also need to improve our Doxygen headers for the TransactionDMA function in general.  Can you improve the adc.h files to fix the following issues?
1. The Doxygen for `mxc_adc_req_t` is broken (no link/documentation generated)
2. Update the `MXC_ADC_StartConversionDMA` function brief to inform the user that they must acquire a DMA channel and pass it in via the "req" input struct
3. Update the `MXC_ADC_StartConversionDMA` function brief to inform the user that they must enable the correct DMA IRQ call the `MXC_DMA_Handler` function from the ISR.